### ISSUE TITLE
New option: alternative font for active window (e.g. bold)

### ISSUE THREAD

**Issue by tuurep** [tuurep/windowlist#24]
Active window in a bold font could look great

It may be doable with:
- bold variant of font specified in polybar's `config.ini`
- optionally adding a `%{T}` tag in windowlist:
  - https://github.com/polybar/polybar/wiki/Formatting#font-t

**Comment by Dupond** [comment]
I don't know if you've already decided how to implement this, and if you've chosen to limit the option to the active window.

It seems to me that adding two options `active_window_format` and `inactive_window_format` (perhaps with three possible values: "italic", "bold" and "bolditalic") would cover all possible user needs, and avoid later complaints :)

**Comment by tuurep** [comment]
As far as I can see basically the only way to do this option is to give nothing but a number in `config.toml`, like so:

```toml
# Use `font-1` from `config.ini`
active_window_alt_font = "2" # Yes, 2. The tag uses 1-based indexing, and `config.ini` font vars 0-based.
```

~~Probably should be a string, to set defaults as "none" and not print the tag at all (in which case `font-0` is used)~~

**Edit:** Will probably set it as a number actually. This way we can use 0-indexing if we want, and set defaults as 0.

Somewhat annoying and confusing, but if this is what we have to do, let's go for it 😄 

I'm planning to add an `-font` key for all elements (similarly to the colors), because the way the code is structured, we pretty much get that 'for free'.

---

In conclusion: I'd really like if you could set the actual font name in the options, but that's not possible. Let's try:

```toml
active_window_font   = 0 # Use `font-0`
inactive_window_font = 1 # Use `font-1`
separator_font       = 2 # Use `font-2`
empty_desktop_font   = 3 # Use `font-3`
overflow_font        = 4 # Use `font-4`
```

**Comment by tuurep** [comment]
Even though the doc example says

```
%{T3} Uses font-2 %{T-} use first (default) font
```

This does not mean I could e.g. only have `font-1` in my `config.ini` and that's the first -- tried that out, first font has to be named  `font-0`

(if the opposite was true, a "none" could be a more practical default, but I think 0 is good)

**Comment by tuurep** [comment]
To be very clear, when `*_font = 0`, this is what actually happens:

> If you don't explicitly set the font-index for a tag, the bar will iterate the font list and pick the first one that is able to draw the character.

Because I made it not print the tag when the option is 0, in PR above.

**Comment by tuurep** [comment]
Usage:

Polybar `config.ini`:

```ini
font-0 = DejaVu Sans
font-1 = DejaVu Sans:weight=bold
```

Windowlist `config.toml`:

```toml
active_window_font = 1
```
